### PR TITLE
Expose exam online meeting link

### PR DESCRIPTION
**Pull request by szymon-romanko**
I've made some changes to list_exams() and _parse_exam() functions,
so now they will also save a link to zoom meeting ( if category of the exam is set to online class )

### CODE REVIEW

File: librus/__init__.py [hunk 1]
```diff
@@ -39,11 +39,11 @@ def list_exams(self):
         # TODO we should be iterating explicitly over links to calendar items' details; doing unstructured "grepping" for now
         for element in response.html.search_all("szczegoly/{}'"):
             details = self._html_session.get(url=f"https://synergia.librus.pl/terminarz/szczegoly/{element[0]}")
-            yield self._parse_exam(details.html.find('table.decorated.small'))
+            yield self._parse_exam(details.html.find('table.decorated.medium'))
```
Comment (findepi) [inline librus/__init__.py:42]: is this change intentional?

**Review by findepi — APPROVED** [review]
thanks for your PR!
LGTM, minor question.

File: librus/__init__.py (hunk 1 above)
Comment (szymon-romanko) [reply librus/__init__.py:42]: yes, afaik they changed the class of the table to "decorated medium center"
"table.decorated.small" didn't work for me

**Comment by findepi** [comment]
merged as 2211243444a802dcac75a406dc0fb86989c8c86a, thanks!

**Comment by findepi** [comment]
Released as 0.0.13. Thanks again!
